client/me/purchases/payment-methods/payment-method-list.tsx

```diff
@@ -69,13 +71,28 @@ class PaymentMethodList extends Component<
 		const paymentMethods = this.props.paymentMethodsState.paymentMethods;
 
 		return (
-			<div className="payment-method-list">
-				<SectionHeader label={ this.props.translate( 'Manage Your Payment Methods' ) }>
-					{ this.renderAddPaymentMethodButton() }
-				</SectionHeader>
+			<>
+				{ isJetpackCloud() && (
```

Comment:
Should we add another `isAgency` check to ensure it is only visible to agency users?

Reply:
Very good point. I have made the changes: https://github.com/Automattic/wp-calypso/pull/86572/commits/623d4bb01d4b20c9d60658ac5447883e9491f9ec